src/lib.rs

```diff
@@ -57,6 +60,9 @@ impl TreeState {
         }
     }
 
+    /// Close a tree node.
+    /// Return `true` if the node was open and has been closed.
+    /// Return `false` if the node was already closed.
```

Comment:
```suggestion
    /// Returns `true` if the node was open and has been closed.
    /// Returns `false` if the node was already closed.
```

Reply:
I have no strong preference, because the intent is clear either way. So it's mostly a stylistic thing.